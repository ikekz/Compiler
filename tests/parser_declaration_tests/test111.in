procedure p(a: integer; p: double);
begin
end;
begin
end.