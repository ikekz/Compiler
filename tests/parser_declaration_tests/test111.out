integer        type           integer
double         type           double
char           type           char
p              procedure      args:
                              a              valueparam     integer
                              p              valueparam     double
                              locals:
                              none
